docs/S04-developer-tooling/M7-web3-frontend/L2-demystifying-connect-wallet/index.md

```diff
@@ -93,7 +93,7 @@ We can see here the connect function will now *attempt* to ask a user to connect
 Next step is to create a variable and assign the `eth_requestAccounts` method to it. See the code below:
 
 ```jsx
-const accounts = (await window.ethereum.request({ method: 'eth_requestAccounts' })[0]`
+const account = (await window.ethereum.request({ method: 'eth_requestAccounts' })[0]`
```

Comment:
plural was misleading since we get only the first account from the array

Reply:
Ooooh, going to update this in the codesandbox example!